pyproject.toml

```diff
@@ -200,3 +200,4 @@ testpaths = "tests"
 # use `pytest-mock` drop-in replacement for `unittest.mock`
 # https://pytest-mock.readthedocs.io/en/latest/configuration.html#use-standalone-mock-package
 mock_use_standalone_module = false
+log_level = "warning"
```

Comment:
This is the default, but by adding it to this config file. It's easier to develop to tweak for local debugging.